app/core/templates/core/challenge_list.html

```diff
@@ -8,9 +8,9 @@ <h1>Challenges</h1>
     <tr>
       <td><a href="{% url 'challenge-detail' pk=challenge.pk %}">{{ challenge }}</a></td>
       {% if challenge.is_active %}
-        <td>OPEN</td>
+        <td style="color:green;"><strong>OPEN</strong></td>
```

Comment:
use a CSS class for this!
```suggestion
        <td class="active-status">OPEN</td>
```
Then add the class in core/static/core/style.css
```
.active-status {
  font-weight: bold;
  color: green;
}
.inactive-status {
  font-weight: bold;
  color: red;
}
```